src/App.js

```diff
@@ -1,25 +1,48 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import Welcome from './components/Welcome.js'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      error: null,
+      isLoaded: false,
+      welcomeString: 'hello'
+    }
+  }
+
+  componentDidMount () {
+    fetch('https://safe-wave-09726.herokuapp.com/api/v1/welcome')
+      .then(res => res.text())
+      .then(
+        (result) => {
+          this.setState({
+            isLoaded: true,
+            welcomeString: result
+          })
+        },
+        (error) => {
+          this.setState({
+            isLoaded: true,
+            error
+          })
+        }
+      )
+  }
+
+  render () {
+    const { error, isLoaded, welcomeString } = this.state
```

Comment:
All of this needs tests. Your webpage can be in three different states depending on whether it is loaded successfully, loading, or failed to load.